.devcontainer/Dockerfile

```diff
@@ -24,27 +24,22 @@ RUN apt-get update \
     && apt-get -y install software-properties-common cron
 RUN LC_ALL=C.UTF-8 add-apt-repository -y ppa:ondrej/php
 RUN apt-key adv --keyserver keyserver.ubuntu.com --recv 4F4EA0AAE5267A6C
-# install apt deps (first row for puppeteer)
-RUN apt-get -y install libnss3 libnspr4 libatk1.0-0 libatk-bridge2.0-0 libcups2 libdrm2 libxkbcommon0 libxcomposite1 libxdamage1 libxfixes3 libxrandr2 libgbm1 libasound2
-RUN apt-get -y install postfix python3-setuptools wget jq sudo git apache2 php${PHP_VERSION} php${PHP_VERSION}-xdebug libapache2-mod-php${PHP_VERSION} pwgen php${PHP_VERSION}-apcu php${PHP_VERSION}-gd php${PHP_VERSION}-xml php${PHP_VERSION}-mbstring zip unzip php${PHP_VERSION}-zip curl php${PHP_VERSION}-curl
-RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
+RUN apt-get -y install postfix python3-setuptools wget jq sudo git php${PHP_VERSION} php${PHP_VERSION}-xdebug pwgen php${PHP_VERSION}-apcu php${PHP_VERSION}-gd php${PHP_VERSION}-xml php${PHP_VERSION}-mbstring zip unzip php${PHP_VERSION}-zip curl php${PHP_VERSION}-curl
+
 # Cleanup
 RUN apt-get -y autoremove && \
     apt-get -y clean
 
 # Add image configuration and scripts
 ADD ${SCRIPTS_DIR_PATH}/post-create.sh /post-create.sh
-RUN chmod 755 /post-create.sh
+RUN chmod +x /post-create.sh
```

Comment:
if you do chmod +x locally and add + commit the change, this line is superfluous.

Reply:
thanks for the feedback... Done making changes